docs/setup/tribe.asciidoc

```diff
@@ -0,0 +1,29 @@
+[[tribe]]
+== Using Kibana with Tribe nodes
+
+Kibana can be configured to connect to a tribe node for data retrieval.  Because tribe nodes can't create indices, Kibana additionally
+requires a separate connection to a node to maintain state.  When configured, searches and visualizations will retrieve data using
+the tribe node and administrative actions (such as saving a dashboard) will be sent to non-tribe node.
+
+[float]
+[[tribe-configuration]]
+=== Configuring Kibana for tribe nodes
+
+Tribe nodes take all of the same configuration options used when configuring elasticsearch in `kibana.yml`.  Tribe options
+are prefixed with `elasticsearch.tribe` and at a minimum requires a url:
+[source,text]
+----
+elasticsearch.url: "<your_administration_node>"
+elasticsearch.tribe.url: "<your_tribe_node>"
+----
+
+The full list of configurations can be found at <<settings>>.
```

Comment:
We should update settings.asciidoc as well to reflect `elasticsearch.tribe.{url,username,ssl,requestHeadersWhitelist,customHeaders,requestTimeout,pingTimeout`}